src/org/traccar/database/DeviceManager.java

```diff
@@ -316,7 +316,16 @@ public void removeGroup(long groupId) throws SQLException {
         groupsById.remove(groupId);
     }
 
-    public String lookupAttribute(long deviceId, String attributeName) {
+    public String lookupServerAttribute(long deviceId, String attributeName) {
```

Comment:
It's better, but it still looks a bit clumsy. What do you think about adding a few helper methods with default values. For example:
```Java
public boolean lookupServerBoolean(long deviceId, String attributeName, boolean defaultValue);
public boolean lookupServerString(long deviceId, String attributeName, String defaultValue);
```
Then we won't need parse string everywhere and check for nulls. Do you think it would be a good idea? I think we just need int, boolean and string as a start.